File: python/05-functions/practice/fp-2.py
n=list(map(int,input().split()))
size=len(n)
if(size<0 or size >20   ):
    print("Invalid input")

else:
    product =1
    for i in n:
        if i>20:
            print("Invalid input")
            exit()
        else:
            product*=i
    print(product)

File: java/6-functions/practice/fp-2.java
import java.util.Scanner;
public class A{
    public static void main(String[] args)
    {
        Scanner sc =  new Scanner(System.in);
        int a = sc.nextInt();
        if(a<0 ||a>1000)
        {
            System.out.println("Invalid input");
            return;
        }
            
        try
        {
        int count =0;
        for(int i = 0;i<a;i++)
        {
            int num =sc.nextInt();
            if(num<0)
                count++;
        }
        System.out.println(count);
        }catch (Exception e)
        {
            System.out.println("Invalid input");
        }
    }
}